src/nuklear_color.c

```diff
@@ -23,6 +23,16 @@ nk_parse_hex(const char *p, int length)
     return i;
 }
 NK_API struct nk_color
+nk_rgb_factor(const struct nk_color col, const float factor)
+{
+    struct nk_color ret;
+    ret.r = col.r * factor;
+    ret.g = col.g * factor;
+    ret.b = col.b * factor;
+    ret.a = col.a;
+    return ret;
```

Comment:
Could have micro performance tweak here :wink:  .... Also, not impacting the alpha is intentional, correct?
``` suggestion
    if (factor == 1.0f)
        return col;
    col.r *= factor;
    col.g *= factor;
    col.b *= factor;
    return col;
```

Reply:
Ahh yes! Good catch!